eng/common/scripts/Add-RetentionLease.ps1

```diff
@@ -19,24 +19,28 @@ param(
   [int]$DaysValid,
 
   [Parameter(Mandatory = $true)]
-  [string]$Base64EncodedAuthToken
+  [string]$AccessToken
 )
 
+$unencodedAuthToken = "nobody:$AccessToken"
+$unencodedAuthTokenBytes = [System.Text.Encoding]::UTF8.GetBytes($unencodedAuthToken)
+$encodedAuthToken = [System.Convert]::ToBase64String($unencodedAuthTokenBytes)
```

Comment:
Should we set the encodedAuthToken as a secret value to ensure it doesn't get dumped out into the logs?

Reply:
Not sure what you mean? Is there a mechanism in PowerShell to declare a variable as as secret so it is masked out? Or are you suggesting emitting a ##vso secret variable so that Azure DevOps will mask it for us?

Reply:
I was thinking about the set variable like we do in other places like https://github.com/Azure/azure-sdk-tools/blob/master/eng/common/TestResources/build-test-resource-config.yml#L22.

Reply:
I've made this change, but I'm not entirely comfortable with it. I think that there is some exposure either way here. Fortunately in this context we are using ```$(System.AccessToken)``` which is relatively short lived.